README.md

```diff
@@ -1,3 +1,3 @@
 # MFWSOGH
-First website that I am coding on GitHub (When finished on my server, will post there)
-This website is to have diffrent catagories (computer, mental health, cats, etc) and will have directories and user submitted items in which an admin or moderator will incorpate into the sections user request them in. If you wish to submit an idea feel free to submit an pull request.
+First website that I am working on using GitHub (when finished on my server, will post on GitHub).
+This website will have different catagories (like computer, mental health, cats, etc), directories and user submitted items where an admin or moderator can incorpate into the sections user request them in. If you wish to submit an idea feel free to submit a pull request.
```

Comment:
change "catagories" to categories 
what is incorpate ? i dont think it is a word in english .